homeassistant/components/maxcube/climate.py

```diff
@@ -95,13 +109,73 @@ def current_temperature(self):
 
     @property
     def hvac_mode(self):
-        """Return current operation (auto, manual, boost, vacation)."""
+        """Return current operation mode."""
+        device = self._cubehandle.cube.device_by_rf(self._rf_address)
+        if device.target_temperature == OFF_TEMPERATURE:
+            return HVAC_MODE_OFF
+        if device.target_temperature == ON_TEMPERATURE:
+            return HVAC_MODE_HEAT
         return HVAC_MODE_AUTO
 
     @property
     def hvac_modes(self):
         """Return the list of available operation modes."""
-        return self._operation_list
+        return [HVAC_MODE_OFF, HVAC_MODE_AUTO, HVAC_MODE_HEAT]
+
+    def set_hvac_mode(self, hvac_mode: str):
+        """Set new target hvac mode."""
+        device = self._cubehandle.cube.device_by_rf(self._rf_address)
+        temp = device.target_temperature
+        mode = device.mode
+
+        if hvac_mode == HVAC_MODE_OFF:
```

Comment:
I think we shouldn't use these tricks if the thermostat doesn't support heat/off mode

Reply:
I think that HVAC_MODE_AUTO and HVAC_MODE_HEAT are enough.

Reply:
Device claims to support On (valves fully open - always heat) and Off modes (valves fully closed - never heat), but they're implemented weirdly, through the magic temperatures. When you set the thermostat to Off (through the dial on the thermostat), the display will show Off, but the reported target temperature through the API will be 4.5 degrees. Same with On, but temperature is 30.5 degrees.

Reply:
So, I've tested this solution in my home environment and I like it.